pkg/security/interfaces.go

```diff
@@ -24,35 +23,13 @@ import (
 	"github.com/Peripli/service-manager/pkg/web"
 )
 
-// AuthenticationDecision represents a Authenticator decision to allow or deny authentication or to abstain from
-// taking a decision
-type AuthenticationDecision int
```

Comment:
should we keep it close to the interfaces (e.g. in the same file and just rename it ? - not that it matters too much, just less files)

Also how about since here we talk about security/interfaces, we expose some interfaces for plugging authz logic?